components/add-event-form.tsx

```diff
@@ -18,20 +18,41 @@ export default function AddEventForm({
 }: AddEventFormProps) {
   const API_KEY = process.env.NEXT_PUBLIC_GOOGLE_API_KEY || "";
   const [venueData, setVenueData] = useState({} as Venue);
+  const [resetField, setResetField] = useState(false);
+  const [open, setOpen] = useState(false);
+
+  const formRef = useRef<HTMLFormElement>(null);
 
   const handleVenue = ({ name, lat, lng }: Venue) => {
     setVenueData({ name, lat, lng });
   };
 
-  const addLocationWithVenue = addLocation.bind(null, venueData);
+  const unsetResetField = () => {
+    setResetField(false);
+  };
+
+  const handleSubmit = async () => {
+    await addLocation.bind(null, venueData);
```

Comment:
This looks weird to me. Are you aware you're not calling the function?

I would think it should be like this:
```suggestion
  const handleSubmit = async (formData: FormData) => {
    await addLocation(venueData, formData);
```